_search-plugins/knn/settings.md

```diff
@@ -25,3 +25,4 @@ Setting | Default | Description
 `knn.model.index.number_of_shards`| 1 | The number of shards to use for the model system index, the OpenSearch index that stores the models used for Approximate Nearest Neighbor (ANN) search.
 `knn.model.index.number_of_replicas`| 1 | The number of replica shards to use for the model system index. Generally, in a multi-node cluster, this should be at least 1 to increase stability.
 `knn.advanced.filtered_exact_search_threshold`| null | The threshold value for the filtered IDs that is used to switch to exact search during filtered ANN search. If the number of filtered IDs in a segment is less than this setting's value, exact search will be performed on the filtered IDs.  
+`knn.faiss.avx2.disabled` | False | Whether to disable the SIMD-based `libopensearchknn_faiss_avx2.so` library and load the default `libopensearchknn_faiss.so` library for the Faiss engine on machines with x64 architecture. For more information, see [SIMD optimization for the Faiss engine]({{site.url}}{{site.baseurl}}/search-plugins/knn/knn-index/#simd-optimization-for-the-faiss-engine).
```

Comment:
same here...I feel it is better not to use default for the non optimized version. Also, pls mention it is a static setting